jobrunner/lib/database.py

```diff
@@ -104,6 +160,35 @@ def select_values(itemclass, column, **query_params):
     return [decode_field_values(fields, row)[0] for row in cursor]
 
 
+def get_flag(name):
```

Comment:
I'm not sure `get_flag` and `set_flag` belong in here, as this is supposed to be generic, schema-agnostic stuff. I think `queries.py` would be the place to put them.

It might also help consistency of the codebase to use the shitty ORM (despite it being shitty) and have `Flag` be a dataclass in `models.py` but I don't feel quite as strongly about that.